src/components/Global/DirectSendQR/index.tsx

```
@@ -1,40 +1,124 @@
+                            redirectUrl = `/${address}@arbitrum/usdc`
+                        }
+                    } catch (error) {
+                        toast.error('Error parsing EIP-681 URL')
+                        Sentry.captureException(error)
+                    }
+                }
+                break
+            case EQrType.ENS_NAME:
+                {
+                    const resolvedAddress = await resolveFromEnsName(data.toLowerCase())
+                    if (!!resolvedAddress) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)